ateam-common-packets/rust-lib/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!("stspin-bindings.rs");
```

Comment:
Should we make this a `pub mod stspin_bindings`? Then everything is nicely contained in it's own module

Reply:
Agreed.

Reply:
Actually should this be `pub mod ateam_common_packets` or otherwise have one more later of hierarchy? Or maybe just `pub mod stspin_packets`